### PR TITLE
[NUI] Guide for Notification enhancement

### PR DESCRIPTION
**Pull request by patrykka**
### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 1]
```diff
@@ -42,7 +48,27 @@ To create a notification using property, follow these steps:
     notification.Post(3000);
     ```
 
-3. Respond to OnWindowTouch:
+3. Set notfication content:
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:51]: Corrected notification spell & introduced "the" article
```suggestion
3. Set the notification content:
```

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 2]
```diff
@@ -42,7 +48,27 @@ To create a notification using property, follow these steps:
...
+    view.Add(text);
+
+    // Add view to main Window
+    Window.Instance.Add(view);
+
+    // Post notification for 3000ms
+    notification.Post(3000);
+    ```
+
+4. Respond to OnWindowTouch:
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:71]: Just a doubt,  `OnWindowTouch` is an action that triggers a particular event, can we keep this in code styling, like this: 

```suggestion 
4. Respond to `OnWindowTouch`:

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@patrykka Request you to incorporate the comments, 
Thanks!

File: docs/application/dotnet/guides/nui/nui-components/Notification.md (hunk 2 above)
Comment (patrykka) [reply docs/application/dotnet/guides/nui/nui-components/Notification.md]: change done, thank you for your comment

File: docs/application/dotnet/guides/nui/nui-components/Notification.md (hunk 1 above)
Comment (patrykka) [reply docs/application/dotnet/guides/nui/nui-components/Notification.md]: change done, thank you for your comment

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 3]
```diff
@@ -13,18 +13,24 @@ To create a notification using property, follow these steps:
 
     ```cs
     View view = new View();
-    notification = new Notification(view);
+    // set notification background color
+    view.Size2D = new Size2D(720, 100);
+    view.Position = new Position(0, 0, 0);
+    view.BackgroundColor = new Color(0, 255, 0, 1.0f);
+
+    Noticication notification = new Notification(view);
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:21]:
```suggestion
    Notification notification = new Notification(view);

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 4]
```diff
@@ -13,18 +13,24 @@ To create a notification using property, follow these steps:
 
     ```cs
     View view = new View();
-    notification = new Notification(view);
+    // set notification background color
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:16]:
```suggestion
    // set notification background color.

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 5]
```diff
@@ -13,18 +13,24 @@ To create a notification using property, follow these steps:
...
+    view.BackgroundColor = new Color(0, 255, 0, 1.0f);
+
+    Noticication notification = new Notification(view);
+
     ```
 
 2. Set the notification property:
 
     ```cs
     // Sets a user-defined animation to play when dismiss the notification.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:28]:
```suggestion
    // Sets a user-defined animation to play when dismissing the notification.

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 6]
```diff
@@ -42,7 +48,27 @@ To create a notification using property, follow these steps:
...
+
+    ```cs
+    // Sets text in Notification
+    TextLabel text = new TextLabel("This is notification sample.");
+    text.HorizontalAlignment = HorizontalAlignment.Center;
+    text.VerticalAlignment = VerticalAlignment.Center;
+    text.TextColor = Color.Black;
+    text.Size2D = new Size2D(720, 100);
+
+    // Add text label to view
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:61]:
```suggestion
    // Add text label to view.

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 7]
```diff
@@ -42,7 +48,27 @@ To create a notification using property, follow these steps:
...
+    view.Add(text);
+
+    // Add view to main Window
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:64]:
```suggestion
    // Add view to the main Window.

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 8]
```diff
@@ -42,7 +48,27 @@ To create a notification using property, follow these steps:
...
+    Window.Instance.Add(view);
+
+    // Post notification for 3000ms
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:67]:
```suggestion
    // Post notification for 3000ms.

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@patrykka some more comments to ensure consistency.
Request you to incorporate the comments. 
Thanks.

**Comment by nawab-samsung** [comment]
@patrykka Gentle reminder: Please incorporate the comments.

**Comment by patrykka** [comment]
All suggested changes applied.

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 9]
```diff
@@ -42,7 +48,27 @@ To create a notification using property, follow these steps:
...
+    view.Add(text);
+
+    // Add view to the main Window.
+    Window.Instance.Add(view);
+
+    // Post notification for 3000ms.
+    notification.Post(3000);
+    ```
+
+4. Respond to `OnWindowTouch`:
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:71]: @patrykka

Is `OnWindowTouch` an event?  or what? 


 4. Respond to `OnWindowTouch`...........???:

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@patrykka : Just one comment, please let me know such that I can suggest an appropriate word to make a sentence more meaningful

**Comment by nawab-samsung** [comment]
@patrykka Request you to kindly clear my doubt? and provide your valuable suggestion. 
https://github.com/Samsung/tizen-docs/pull/1355#discussion_r531389809

File: docs/application/dotnet/guides/nui/nui-components/Notification.md (hunk 9 above)
Comment (patrykka) [reply docs/application/dotnet/guides/nui/nui-components/Notification.md:71]: Yes, this is event.
Respond to OnWindowTouch event

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 10]
```diff
@@ -42,7 +48,27 @@ To create a notification using property, follow these steps:
     notification.Post(3000);
     ```
 
-3. Respond to OnWindowTouch:
+3. Set the notification content:
+
+    ```cs
+    // Sets text in Notification
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:54]: Period missing and n lower case
// Sets text in notification

or

 // Sets text in notification

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 11]
```diff
@@ -42,7 +48,27 @@ To create a notification using property, follow these steps:
...
+    TextLabel text = new TextLabel("This is notification sample.");
+    text.HorizontalAlignment = HorizontalAlignment.Center;
+    text.VerticalAlignment = VerticalAlignment.Center;
+    text.TextColor = Color.Black;
+    text.Size2D = new Size2D(720, 100);
+
+    // Add text label to view.
+    view.Add(text);
+
+    // Add view to the main Window.
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:64]: window lower case?

File: docs/application/dotnet/guides/nui/nui-components/Notification.md (hunk 9 above)
Comment (mijin85cho) [reply docs/application/dotnet/guides/nui/nui-components/Notification.md:71]: @patrykka To this dialogue I have additional questions.

It seems the `OnWindowTouch` is defined event, but in this page defining is not written.

Also when I checked in the API reference, https://samsung.github.io/TizenFX/API8/api/Tizen.NUI.Components.Notification.html,
I couldn't find SetDismissOnTouch method.

Would you please check?

File: docs/application/dotnet/guides/nui/nui-components/Notification.md (hunk 10 above)
Comment (nawab-samsung) [reply docs/application/dotnet/guides/nui/nui-components/Notification.md:54]: @patrykka 
```suggestion 
    // Sets text in notification.

File: docs/application/dotnet/guides/nui/nui-components/Notification.md (hunk 11 above)
Comment (nawab-samsung) [reply docs/application/dotnet/guides/nui/nui-components/Notification.md:64]:
```suggestion 
    // Add view to the main window.

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@patrykka Request you to incorporate the comments and also please answer our doubts. 
Thanks

File: docs/application/dotnet/guides/nui/nui-components/Notification.md (hunk 9 above)
Comment (nawab-samsung) [reply docs/application/dotnet/guides/nui/nui-components/Notification.md:71]: Once you respond to @mijin85cho query, then please incorporate the following (if it is still an event).
```suggestion 
4. Respond to `OnWindowTouch` event: 
```

**Comment by nawab-samsung** [comment]
@patrykka Request you to incorporate the comments.

**Comment by nawab-samsung** [comment]
@patrykka Gentle reminder:: Request you to incorporate the comments.

File: docs/application/dotnet/guides/nui/nui-components/Notification.md (hunk 9 above)
Comment (patrykka) [reply docs/application/dotnet/guides/nui/nui-components/Notification.md:71]: @mijin85cho  @nawab-samsung 
 OnWindowTouch is Window event - screen ( I left that from previous description, while in example  -> https://github.com/Samsung/Tizen-CSharp-Samples/pull/194 I used some kind of "timer" to show notification on each 5 seconds.

Regarding SetDismissOnTouch  - looks this is public function with marked "[EditorBrowsable(EditorBrowsableState.Never)]"
This mean that it is not published in Components API reference. What I should do with it ?

Comment (mijin85cho) [reply docs/application/dotnet/guides/nui/nui-components/Notification.md:71]: OnWindowTouch is Window event - screen : 
It seems "OnWindowTouch" is not a specific function or method with name "OnWindowTouch", but just mentioning an event notion.
If so, please remove the code tag. This line seems to mean "Respond to the touch event on window", not "Respond to an event with a name, OnWindowTouch "


Regarding SetDismissOnTouch : 
Notification guide will be published on docs.tizen.org on public. 
So if there is a non published method to the public, please don't use it.
I'm concerned about the case that developers in outside try to follow the guide but can't find that method.

If I understood something wrong, please let me know.

Comment (patrykka) [reply docs/application/dotnet/guides/nui/nui-components/Notification.md:71]: ok, this point was related only with dismissing notification, point 4 deleted

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 12]
```diff
@@ -13,18 +13,24 @@ To create a notification using property, follow these steps:
 
     ```cs
     View view = new View();
-    notification = new Notification(view);
+    // set notification background color.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:16]: To maintain consistency: 
```suggestion 
    // Sets notification background color.

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 13]
```diff
@@ -33,7 +39,7 @@ To create a notification using property, follow these steps:
     notification.SetLevel(NotificationLevel.Base);
   
     // Sets position and size of the notification window.
-    positionSize = new Rectangle(0, 0, 600, 300);
+    Rectangle positionSize = new Rectangle(0, 0, 600, 300);
     notification.SetPositionSize(positionSize);
 
     // Post a notification window with the content view.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:45]:
```suggestion 
    // Posts a notification window with the content view.

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 14]
```diff
@@ -42,13 +48,26 @@ To create a notification using property, follow these steps:
...
-    bool dismissOnTouch = true;
-    notification.SetDismissOnTouch(dismissOnTouch);     // Dismiss notification window on touch if the value is true.
+    // Sets text in notification.
+    TextLabel text = new TextLabel("This is notification sample.");
+    text.HorizontalAlignment = HorizontalAlignment.Center;
+    text.VerticalAlignment = VerticalAlignment.Center;
+    text.TextColor = Color.Black;
+    text.Size2D = new Size2D(720, 100);
+
+    // Add text label to view.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:61]:
```suggestion    
    // Adds text label to view.

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 15]
```diff
@@ -42,13 +48,26 @@ To create a notification using property, follow these steps:
...
+    view.Add(text);
+
+    // Add view to the main window.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:64]:
```suggestion 
    // Adds view to the main window.

File: docs/application/dotnet/guides/nui/nui-components/Notification.md [hunk 16]
```diff
@@ -42,13 +48,26 @@ To create a notification using property, follow these steps:
...
+    Window.Instance.Add(view);
+
+    // Post notification for 3000ms.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/nui-components/Notification.md:67]:
```suggestion 
    // Posts notification for 3000ms.

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@patrykka Minor comments, please incorporate. 
Following which, shall approve. 
Thanks

**Review by nawab-samsung — APPROVED** [review]
@patrykka Thanks for incorporating the comment.

**Pull request merged**